.github/workflows/software-environments.yml

```diff
@@ -0,0 +1,43 @@
+name: Create software environments
+
+on:
+  push:
+    branches: main
```

Comment:
Since we won't need to run this workflow very often (normally only when we issue a new `coiled-runtime` release), and the build matrix below creates several CI jobs, we could use [GHAs `path` filtering](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) to only run this workflow when a file in `environments/` is added / removed / modified. It should look something like

```yaml
on:
  push:
    branches: main
    paths:
      - environments/**
```

Reply:
I like this.